packages/loaders/src/Loader.ts

```diff
@@ -668,9 +631,8 @@ class Loader
     private static _shared: Loader;
     /**
      * If this loader cannot be destroyed.
-     * @default false
      */
-    private _protected: boolean;
+    private _protected = false;
```

Comment:
This is technically a code change.

Reply:
Should I change it back? Or is there something else I need to do?

Reply:
@omer8374 if you could change and it back and push, that would be great. Thanks. In general, we want these documentation fixes to not touch any code, even if it seems small and trivial.

Reply:
@bigtimebuddy thanks I changed it back.